<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="asset_no">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>851</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>831</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Asset Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="brd"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="srno"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Serial No.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="model"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Brand</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QComboBox" name="ast_typ">
       <item>
        <property name="text">
         <string>-- Select --</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Manufacturer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="man"/>
     </item>
     <item row="2" column="7">
      <widget class="QComboBox" name="status">
       <item>
        <property name="text">
         <string>-- Select --</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In Use</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In Stock</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QDateEdit" name="purdate"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="loc">
       <item>
        <property name="text">
         <string>-- Select --</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="desc"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Asset Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Vendor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLineEdit" name="vendor">
       <property name="placeholderText">
        <string>(Optional)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Purchase Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>PO No.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cndtn">
       <item>
        <property name="text">
         <string>-- Select --</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Working</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Good</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fair</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poor</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Install Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Condition</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="pono">
       <property name="placeholderText">
        <string>(Optional)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Warranty Expire</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="10">
      <widget class="QDateEdit" name="instdate"/>
     </item>
     <item row="6" column="9" colspan="2">
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="7" column="9" colspan="2">
      <widget class="QPushButton" name="reset_btn">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Allocated To</string>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QLineEdit" name="alloc"/>
     </item>
     <item row="2" column="10">
      <widget class="QDateEdit" name="wrnty"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
